Source/Extensions/Blazorise.Components/Autocomplete.razor

```diff
@@ -46,10 +46,6 @@
 
         @if ( DropdownVisible )
         {
-            // once the menu is opened we need to register it as closable so that user could properly close it with escape key
-            // without accidentally triggering the parent component closing
-            RegisterClosableComponent();
```

Comment:
While it might seem a good and harmfull change. Our Closable component registering has a reason if I remember correctly. We can have multiple closable components registered and this makes sure ESC key, loose focus, etc work properly.

I'll have to remember some use cases to see if they still work fine.
Maybe `Modal` + `AutoComplete`, will the ESC Key still have the correct behaviour?

Reply:
As mentioned, unfortunately using closable as is make the autocomplete look buggy. Closable should not fire close() when we click anywhere in the dropdown, including scrollbar

Reply:
So the problem was with the new scroll feature and Closable right? Because so far it was working without it I guess.